src/Sylius/Component/Core/Formatter/StringInflector.php

```diff
@@ -20,7 +20,7 @@ public static function nameToCode(string $value): string
         return str_replace([' ', '-', '\''], '_', $value);
     }
 
-    public static function nameToSlug(string $value): string
+    public static function codeToSlug(string $value): string
```

Comment:
why did you change it?
is is a BC? If it is you should add a new method :)

Reply:
It was not used anywhere, and was not consistent, but you aright, it can be BC break.